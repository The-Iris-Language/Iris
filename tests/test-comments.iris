.. test for comments ignored by scanner
class Main () { 
  int main() {
    .. COMMENT
    .~* Val is 
    da tester *~.
    .~* da goat *~.
    char c;
    c = 'a';
    c = '\r';
    return 0;
  }
}