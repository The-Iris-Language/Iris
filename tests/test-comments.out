class Main of Object () { 
int main() {
char c;
c = 'a';
c = '\r';
return 0;
}
}